platform-sdk/swirlds-merkle/src/test/java/com/swirlds/merkle/test/MerkleSynchronizationTests.java

```diff
@@ -59,19 +57,19 @@
 @Tag(TIMING_SENSITIVE)
 @DisplayName("Merkle Synchronization Tests")
 public class MerkleSynchronizationTests {
-    private final Configuration configuration = new TestConfigBuilder().getOrCreateConfig();
+
+    private final Configuration configuration = new TestConfigBuilder()
+            // This is important! A low value will cause a failed reconnect to finish more quicly.
```

Comment:
Nit: s/quicly/quickly/

Reply:
Good catch! I'll fix the typo in one of my next reconnect fixes.